src/app/components/case/tasks/AddTask/AddTaskForm.tsx

```diff
@@ -1,27 +1,41 @@
 import { FC } from "react"
-import { ScaffoldForm } from "@amsterdam/amsterdam-react-final-form"
 
-import ScaffoldFields from "app/components/shared/Form/ScaffoldFields"
+import ConfirmScaffoldForm from "app/components/shared/ConfirmScaffoldForm/ConfirmScaffoldForm"
 import scaffold from "./scaffold"
+import styled from "styled-components"
 
 type Props = {
-  caseId: Components.Schemas.Case["id"]
+  id: Components.Schemas.Case["id"]
 }
 
-const addTask = (task: string, caseId: Components.Schemas.Case["id"]) => {
-  console.log(`taak ${ task } toegevoegd aan ${ caseId }`)
+// Scaffolded label is needed for ConfirmFields
+const Div = styled.div`
+  label {
+    display: none;
```

Comment:
For accessibility reasons I would recommend the CSS clip technique: https://webaim.org/techniques/css/invisiblecontent/#techniques